core/monitor/price/snapshot.go

```diff
@@ -174,15 +174,15 @@ func (e *Engine) serialisePriceRanges() []*types.PriceRangeCache {
 		})
 	}
 
-	sort.Slice(prc, func(i, j int) bool {
+	sort.SliceStable(prc, func(i, j int) bool {
 		if prc[i].Bound.UpFactor.Equal(prc[j].Bound.UpFactor) {
 			if prc[i].Bound.DownFactor.Equal(prc[j].Bound.DownFactor) {
-				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon
+				return prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon && prc[i].Bound.Active == prc[j].Bound.Active
```

Comment:
that doesn't seem correct. shouldn't it be:

``
if prc[i].Bound.Trigger.Horizon == prc[j].Bound.Trigger.Horizon {
        return prc[i].Bound.Active == prc[j].Bound.Active
} 
return  prc[i].Bound.Trigger.Horizon < prc[j].Bound.Trigger.Horizon